README.md

```diff
@@ -4,13 +4,13 @@ This is a project to build and publish docker images for various [Network Optix]
 
 ## License
 
-Licensed under the [MIT License][license].  
+Licensed under the [MIT License][license].

```

Comment:
Please revert the intentional double space at end of lines for newline behaviors.  
Probably related to the remove spaces at end of lines on save option, should be disabled for e.g. markdown and text files.

E.g. workspace settings:

```json
		"files.trimTrailingWhitespace": true,
		"[markdown]": {
			"files.trimTrailingWhitespace": false
		},
		"[plaintext]": {
			"files.trimTrailingWhitespace": false
		},
		"diffEditor.ignoreTrimWhitespace": false,
		"editor.renderWhitespace": "boundary"
```

Reply:
My pleasure, very cool and organized project you have here.
Reverted README changes, operated on README after opening the proper workspace in VSCode and the trailing whitespace was not removed (see latest diff).